livekit/room.py

```diff
@@ -64,6 +70,7 @@ def name(self) -> str:
     def metadata(self) -> str:
         return self._info.metadata
 
+    @property
     def e2ee_manager(self) -> E2EEManager:
```

Comment:
would this change the interface? let's document in changelog

Reply:
👍